scripts/common.sh

```diff
@@ -132,6 +132,11 @@ _copyrights() {
   if [[ $copyright_error -eq 1 ]]; then _error "Wrong Copyright(s)"; fi
 }
 
+_go_license() {
+  go get github.com/google/go-licenses
+  go-licenses check ./...
```

Comment:
According to the docs this should be

```suggestion
  go-licenses check github.com/solarwinds/snap-plugin-lib
```

Have you verified this does the same job for us? I mean if we're sure that an imported lib with "disallowed" license will really be caught buy this

Reply:
yes, I've checked but in such case you need to provide packages names also, so multiples entries would be needed. With ./... it scans everything.